Package.swift

```diff
@@ -0,0 +1,10 @@
+import PackageDescription
+
+let package = Package(
+  name: "HackersAtCambridgeWebsite",
+  dependencies: [
+      .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 6),
```

Comment:
Is this non-uniform indentation a Swiftism? This line is 4 spaces from the previous level, but the previous line is 2.